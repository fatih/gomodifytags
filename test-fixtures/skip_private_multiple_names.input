package foo

type foo struct {
	Bar    string
	t, Baz bool
}
