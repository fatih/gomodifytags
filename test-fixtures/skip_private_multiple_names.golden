package foo

type foo struct {
	Bar    string `json:"bar"`
	t, Baz bool   `json:"baz"`
}
